conda-env/dev.yml

```diff
@@ -0,0 +1,31 @@
+# Conda pcmdi_metrics development environment
+name: pcmdi_metrics_dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  # Base
+  # ==================
+  - python=3.8.10
+  - pip=21.2.4
+  - numpy=1.21.3
+  - matplotlib=3.4.3
+  - cdat::vcs=8.2.1
+  - cdat::vcsaddons=8.2.1
+  - cdms2=3.1.5
+  - genutil=8.2.1
+  - cdutil=8.2.1
+  - cdp=1.7.0
+  - PCMDI::cia=0.0.6
+  - proj4=6.1.1
+  # Testing
+  # ==================
+  - pytest=6.2.5
+  - pytest-cov=3.0.0
+  # Developer Tools
+  # ==================
+  - jupyterlab=3.2.1
+  - nb_conda=2.2.1
+  - nb_conda_kernels=2.3.1
+
+prefix: /opt/miniconda3/envs/pmcdi_metrics_dev
```

Comment:
@acordonez I'm aware that you're creating a new `meta.yaml` for conda-forge. Let me know if these packages are still consistent with your latest `meta.yaml`.

Reply:
@tomvothecoder We axed cdat::vcs, cdat::vcsaddons, and PCMDI::cia. Other packages look right.

Reply:
Great. After removing those packages, `python=3.9.7` now works.

Reply:
@tomvothecoder Some codes in PMP uses `MV2`, should it be included?